src/app/shared/stache/modules/tutorial/tutorial.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-tutorial-container">
```

Comment:
Classname should be the same as the component selector, if possible (`.stache-tutorial`).